extensions.json

```
@@ -346,6 +346,10 @@
       "version": "3.0.39",
       "checkout": "v3.0.39"
     },
+    {
+      "id": "hashicorp.vscode-terraform",
+      "repository": "https://github.com/hashicorp/vscode-terraform"
```

Actually, I'm wondering if it wouldn't be better to publish only a specific version and git tag, instead of always publishing from the default branch. Something like this would seem a bit safer to me:

```suggestion
      "repository": "https://github.com/hashicorp/vscode-terraform",
      "version": "2.1.1",
      "checkout": "v2.1.1"
```